dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -159,7 +159,7 @@ http2.maxConcurrentStreams=Max concurrent streams
 http2.maxConcurrentStreams.desc=Specifies the maximum number of streams that an HTTP/2 connection can have active at any given point. Opening streams over the limit, will result on a REFUSED_STREAM (0x7). If not specified, the default value of concurrent streams will be set to 200.
 
 http2.maxFrameSize=Max frame size
-http2.maxFrameSize.desc=Specifies the maximum allowed size of a frame payload the server will advertise in the SETTINGS_MAS_FRAME_SIZE HTTP/2 settings frame. This can be configured to any size within the range of 16,384 to 16,777,215 bytes, inclusive. If not specified, the default is set to 57,344 bytes.
+http2.maxFrameSize.desc=Specifies the maximum allowed size of a frame payload the server will advertise in the SETTINGS_MAX_FRAME_SIZE HTTP/2 settings frame. This can be configured to any size within the range of 16,384 to 16,777,215 bytes, inclusive. If not specified, the default is set to 57,344 bytes.
```

Comment:
Suggested changes:
- Change "payload the" to "payload that the" (add "that")
- Optional: Change "will advertise" to "advertises"
- Change "This can be configured to any size" to "The size must be"
- Remove ", inclusive"

Reply:
HI Pam, this change was to just update the incorrect setting name, so I'll ignore the other changes!

Thanks for taking a look.